src/api/wakanda.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: process.env.REACT_APP_BACKEND_URL,
+  params: {
+    api_key: 'b81a990598f52450eb9f9ab81027dc56',
```

Comment:
@dusmel What is this API key?

Reply:
This was a placeholder for the token but since I am not using it directly let me remove it for now